docs/apiimplementations.rst

```diff
@@ -0,0 +1,88 @@
+API Implementations
+===================
+
+Bleak hides most of the details of the Bluetooth LE stack provided by the operating system,
+but sometimes it may be useful to access those details because it provides useful functionality 
+(even though your code may no longer be portable to other operating systems).
```

Comment:
We should also mention that the backends do not have a stable API.

Reply:
ok.